src/ARFset.js

```diff
@@ -204,7 +204,7 @@ export default class ARFset {
         // url doesn't need to be a valid url. Extensions to make it valid will be added here
         console.log(urlOrData);
         const targetPrefix = '/markerNFT_' + this.markerNFTCount++
-        const extensions = ['fset', 'iset', 'fset3']
+        const extensions = ['iset', 'fset3', 'fset']
```

Comment:
normally loading a  NFT marker we need a different order 'fset', 'iset', 'fset3' but i think because the FileReader it rearrange the order in this way: ['iset', 'fset3', 'fset'] as in the commit.